selfdrive/updated.py

```diff
@@ -72,15 +76,14 @@ def graceful_shutdown(self, signum, frame):
 
   def update_now(self, signum, frame):
     cloudlog.info("caught SIGHUP, running update check immediately")
+    global DEBUG_FORCE_UPDATE
+    DEBUG_FORCE_UPDATE = True
```

Comment:
FIXME: testing-only configuration. Before merging into master, this line should be commented out, and commits 103a244 and 7f8cca6 should be reverted as well.